render.py

```diff
@@ -41,8 +41,8 @@ def read_pipe(self, pipe, queue):
             queue.put((None, None))
 
     def run(self, command):
-        p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-                             creationflags=subprocess.CREATE_NO_WINDOW)
+        p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```

Comment:
Pretty sure that CREATE_NO_WINDOW is required on Windows, try testing this on Linux:
```suggestion
if os.name == "nt":
    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
                         creationflags=subprocess.CREATE_NO_WINDOW)
else:
    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```